pkg/controller/multi/notary_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Portieris Authors.
+// Copyright 2018,2020 Portieris Authors.
```

Comment:
Please use make copyright to generate this

Reply:
separately fixed the tools

Reply:
#110